src/megaclient.cpp

```diff
@@ -4094,7 +4101,11 @@ bool MegaClient::procsc()
                             useralerts.begincatchup = true;
                         }
                     }
-                    app->catchup_result();
+
+                    if (statecurrent)
```

Comment:
```suggestion
                    if (statecurrent && !insca_notlast)
```
This event should be triggered only if we are actually up to date, so I think it should also check that there's no more pending actionpackets.

Reply:
I agree - thanks very much!